WordPress/gutenberg
# DataForm: remove `FormFieldVisibility`

## What?

This removes a piece of code that is not used, we forgot removing it in a past refactoring.

## Testing Instructions

Build the project, load the Pages page, open the QuickEdit experiment and verify nothing breaks.
